src/bcf-toolkit/Converter/Bcf21/Converter.cs

```diff
@@ -47,6 +47,30 @@ public class Converter : IConverter {
     await WriteJson(target, bcf);
   }
 
+  /// <summary>
+  ///   The method parses the BCF file of version 2.1 and writes into JSON.
+  ///   The root of the BCF zip contains the following files:
+  ///   - project.bcfp (optional)
+  ///   - bcf.version
+  ///   Topic folder structure inside a BCFzip archive:
+  ///   - markup.bcf
+  ///   Additionally:
+  ///   - Viewpoint files (BCFV)
+  ///   - Snapshot files (PNG/JPEG)
+  ///   - Bitmaps
+  /// </summary>
+  /// <param name="source">The path to the BCFzip.</param>
+  /// <param name="target">The target path where the JSON is written.</param>
+  public async Task BcfToJson(string source, string target) {
+    try {
+      await using FileStream fileStream = new FileStream(source, FileMode.Open, FileAccess.Read);
```

Comment:
`var` can be used instead of specifying the type

Reply:
thx, fixed